### ISSUE TITLE
Positioning not handled close to the right edge of the screen

### ISSUE THREAD

**Issue by PlkMarudny** [stordahl/svelte-right-click#5]
While called at the very right edge of the screen, the menu is being drawn partially off-screen, guess it should be rather repositioned in such a case.

**Comment by stordahl** [comment]
Yes I've noticed this! Will address this before v1. Thanks for submitting an issue!
